README.md

```diff
@@ -12,32 +12,73 @@ U.S. Government retains certain rights in this software.
+Welcome to `pyttb`, a refactor of the 
+[Tensor Toolbox for MATLAB](https://www.tensortoolbox.org) in Python.
 
-**Tensor Classes:**
-* `tensor`: dense tensors
-* `sptensor`: sparse tensors
-* `ktensor`: Kruskal tensors
-* `tenmat`: matricized tensors
-* `ttensor`: Tucker tensors
+This package contains data classes and methods for manipulating dense, 
+sparse, and structured tensors, along with algorithms for computing 
+low-rank tensor decompositions:
 
-**Tensor Algorithms:**
-* `cp_als`, `cp_apr`: Canonical Polyadic (CP) decompositions
-* `tucker_als`: Tucker decompostions
+- Data Classes: 
+[`tensor`](https://pyttb.readthedocs.io/en/stable/tensor.html "dense tensors"), 
+[`sptensor`](https://pyttb.readthedocs.io/en/stable/sptensor.html "sparse tensors"), 
+[`ktensor`](https://pyttb.readthedocs.io/en/stable/ktensor.html "Kruskal tensors"), 
+[`ttensor`](https://pyttb.readthedocs.io/en/stable/ttensor.html "Tucker tensors"), 
+[`tenmat`](https://pyttb.readthedocs.io/en/stable/tenmat.html "matricized dense tensors"), 
+[`sptenmat`](https://pyttb.readthedocs.io/en/stable/sptenmat.html "matricized sparse tensors"), 
+[`sumtensor`](https://pyttb.readthedocs.io/en/stable/sumtensor.html "implicit sum of tensors")
+- Algorithms:
+[`cp_als`](https://pyttb.readthedocs.io/en/stable/cpals.html "CP decomposition via Alternating Least Squares"),
+[`cp_apr`](https://pyttb.readthedocs.io/en/stable/cpapr.html "CP decomposition via Alternating Poisson Regression"), 
+[`gcp_opt`](https://pyttb.readthedocs.io/en/stable/gcpopt.html "Generalized CP decomposition"), 
+[`hosvd`](https://pyttb.readthedocs.io/en/stable/hosvd.html "Tucker decomposition via Higher Order Singular Value Decomposition"),
+[`tucker_als`](https://pyttb.readthedocs.io/en/stable/tuckerals.html "Tucker decompostion via Alternating Least Squares")
 
-# Getting Started
-For full details see our [documentation](https://pyttb.readthedocs.io).
 ## Quick Start
-We are on pypi
-```commandline
-pip install pyttb
-```
-or install from source
+
+### Installation
 ```commandline
-pip install .
+python3 -m pip install pyttb
 ```
 
-# Contributing
-Check out our [contributing guide](CONTRIBUTING.md).
+### Example
+```python
+>>> import pyttb as ttb
+>>> X = ttb.tenrand((2,2,2))
+>>> type(X)
+<class 'pyttb.tensor.tensor'>
+>>> M = ttb.cp_als(X, rank=1)
+CP_ALS:
+ Iter 0: f = 7.367245e-01 f-delta = 7.4e-01
+ Iter 1: f = 7.503069e-01 f-delta = 1.4e-02
+ Iter 2: f = 7.508240e-01 f-delta = 5.2e-04
+ Iter 3: f = 7.508253e-01 f-delta = 1.3e-06
+ Final f = 7.508253e-01
+ ```
+
+### Getting Help
+- [Documentation](https://pyttb.readthedocs.io)
+- [Tutorials](https://pyttb.readthedocs.io/en/stable/tutorials.html)
+- [Info for users coming from MATLAB](https://pyttb.readthedocs.io/en/stable/for_matlab_users.html)
+- Learn about tensor decompositions: 
+<!-- markdown-link-check-disable -->
+[tensor paper](https://doi.org/10.1137/07070111X "Tensor Decompositions and Applications by Tamara G. Kolda, Brett W. Bader"), 
```

Comment:
Workaround: The checking provided by `gaurav-nelson/github-action-markdown-link-check` used in [one of the GitHub actions in pyttb](https://github.com/sandialabs/pyttb/blob/main/.github/workflows/markdown-check.yml) breaks on the DOI URL. That action allows for disabling/enabling the checks but does not interact with markdown formatting well, IMHO. I moved the disable/enable marks to before/after this list, but then all of the URLs go unchecked. If I remove the disable for the DOI, the check reports a 403 error (due to the redirects required to resolve to the paper?), and then the check fails.

@ntjohnson1 Any suggestions on how to resolve this more gracefully and still allow the URL checks?

Reply:
I picked that package somewhat arbitrarily based on perceived popularity when we had a bunch of dead links. I'm fine with opting out a variety of links of that makes maintenance easier. Since it defaults to checking at least we know it's our own fault if we miss something.

That author of that package came out with a newer one they recommend https://github.com/UmbrellaDocs/linkspector
We could put an issue at low priority to revisit this. The goal was to reduce burden on checking links, but false positives negate that benefit

Reply:
Yes, I saw the updated GH action from that author. I'll merge this and create a new Issue for the follow-up. Thx.

Reply:
It's possible using the version that ignores the current line would play nicer. But up to you on how valuable you find that experiment.

https://github.com/gaurav-nelson/github-action-markdown-link-check?tab=readme-ov-file#disable-check-for-some-links

Reply:
Note, this check helped fix a lot of URLs I broke along the way on this README refactor, so I am happy to have something like this checking as we dev.